.changeset/khaki-seahorses-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'playroom': minor
+---
+
+Optimise Playroom UI performance.
```

Comment:
I wouldn't really call out this as a perf change. Maybe it _is_ just worth saying something like `Use CSS gap for spacing` or something.

I _do_ think it's worth adding a note to the README saying that Playroom will only ever utilize features supported by all browers, but it assumes you have the latest stable version of any of the major browsers.